src/microbe_stage/Microbe.cs

```diff
@@ -1248,6 +1384,30 @@ internal void SuccessfulKill()
             TranslationServer.Translate("SUCCESSFUL_KILL"));
     }
 
+    private void OnIGotAddedToColony()
```

Comment:
I think that new name might be more confusing because there's already `OnColonyMemberAdded`. I think this whole add and remove stuff needs a refactor pass to make it more clear which objects needs to do what. So I don't think this change is worth it.